_includes/configuration.md

```diff
@@ -1,14 +1,14 @@
 ## Configuring {{ site.data.product.title_short }}
 
-After installing {{ site.data.product.title_short }} and running it for the first
+After installing {{ site.data.product.title_short }} and run it for the first
 time, you must perform some basic configuration. To configure
-{{ site.data.product.title_short }}, you must at a minimum:
+{{ site.data.product.title_short }}, you must complete these steps:
 
-1.  Add a disk to the infrastructure hosting your appliance.
+1.  Add a disk to the infrastructure that is hosting your appliance.
 
 2.  Configure the database.
 
-Configure the {{ site.data.product.title_short }} appliance using the internal
+Configure the {{ site.data.product.title_short }} appliance by using the internal
 appliance console.
```

Comment:
```suggestion
Configure the appliance by using the internal appliance console.
```

Reply:
@bdunne Since the title_short was there previously, I don't think this suggestion should hold up a merge.  WDYT?

Reply:
No, this alone shouldn't hold up a merge